.github/workflows/test.yml

```diff
@@ -8,15 +8,23 @@ on:
       - master
 
 jobs:
-  test_linux:
-    name: "linux test: ${{ matrix.arch }}"
-    runs-on: ubuntu-latest
+  test:
+    name: "Test: Python ${{ matrix.python-version }} on ${{ matrix.os }}"
     strategy:
-      fail-fast: true
       matrix:
-        arch: [aarch64, armhf, armv7, amd64, i386]
```

Comment:
`arch` does not have any effect in the old version, except triggering 5 identical jobs.